public/index.html

```diff
@@ -17,59 +18,62 @@ <h1 class="header__wordmark">
           fluent
         </h1>
         <nav class="header__nav">
-          <a class="link" href="http://projectfluent.io/fluent/guide">Syntax Guide</a>
-          <a class="link" href="http://projectfluent.io/play">Online Editor</a>
-          <a class="link" href="https://github.com/projectfluent">GitHub</a>
-          <a class="link" href="https://groups.google.com/forum/#!forum/mozilla.tools.l10n">Mailing list</a>
-          <a class="link" href="https://github.com/projectfluent/fluent/wiki">Wiki</a>
+          <a class="link" href="http://projectfluent.io/fluent/guide">Learn</a>
+          <a class="link" href="http://projectfluent.io/play">Play</a>
+          <a class="link" href="#implementation-docs">Use</a>
+          <a class="link" href="https://github.com/projectfluent">Contribute</a>
+          <!-- <a class="link" href="https://groups.google.com/forum/#!forum/mozilla.tools.l10n">Community</a> -->
         </nav>
       </header>
 
       <section class="section section--centered">
-        <p class="pitch">
-
-          <strong class="highlight">Fluent</strong> is a localization paradigm
-          designed to unleash the expressive power of the natural language.
-
-        </p>
+        <div class="pitch-wrapper">
+          <p class="pitch">
+            <strong>Fluent</strong> is a set of localization libraries<br>
+            designed to create better translations.
+          </p>
+          <a class="link use" href="#implementation-docs">See how to use it.</a>
+        </div>
 
-        <a class="link--cta" href="https://github.com/projectfluent">Visit GitHub →</a>
+        <div class="cta-wrapper">
+          <a class="link--cta" href="https://www.npmjs.com/package/fluent">Install fluent.js 0.3.1</a>
+          <a class="link other" href="https://github.com/projectfluent/">Other implementations</a>
+        </div>
       </section>
 
       <section class="section">
         <div class="features">
-          <h2 class="features__title">For localizers</h2>
+          <h2 class="features__title">Featuring</h2>
```

Comment:
I also don't agree that this should just be one list. Even if the primary target audience are developers, these developers have to put their localizers' hats when they write copy in the source code. And if they don't, then they won't care about "Grammatical cases"! In which case it's better to just let them skip the list aimed at localizers and get straight to the technical details.